subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -360,9 +367,20 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
             """
         }
 
+        setupLocalBuildCache()
         setupJavaProject()
     }
 
+    private void setupLocalBuildCache() {
```

Comment:
❔ Don't we have anything for smoke tests, yet?

Reply:
Hmm, I didn't find anything there 🤔 .

I can move that method to super class